agent/src/agent/validator/proxy.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+from agent import proxy
+
+
+def is_valid(proxy_obj: proxy.Proxy) -> bool:
+    try:
+        requests.get('http://example.com', proxies=proxy.get_config(proxy_obj), timeout=5)
```

Comment:
Can we check here `https://<destination_url>/api/v1/status`? Because some clients might whitelist only specific anodot ips

Reply:
we'll check in both places